doc/it.xsemantics.doc/contents/00-Main.html

```diff
@@ -1,3 +1,14 @@
+<!--
```

Comment:
iirc, we had a lot of discussions with the IP peeps about copyright headers in generated artifacts. If it is possible, they want to have them.

Does xdoc have a feature for adding copyright headers?